packages/account/src/Configs/address-details-config.js

```diff
@@ -16,7 +16,7 @@ const address_details_config = ({ account_settings, is_svg }) => {
                 [
                     'regular',
                     localize(
-                        'Only letters, numbers, periods, hyphens, apostrophes, and spaces, please.'
+                        'Use only the following special characters: . \” , \' ; ( ) @ # / -'
```

Comment:
the double quote mentioned here `"` is not looking right, especially since it is skewed `”` instead of `"`.

Reply:
Thanks, replaced quotes here.